velox/common/base/Exceptions.h

```diff
@@ -172,14 +172,16 @@ std::string errorMessage(fmt::string_view fmt, const Args&... args) {
         exception,                                                       \
         typename ::facebook::velox::detail::VeloxCheckFailStringType<    \
             decltype(message)>::type>(veloxCheckFailArgs, message);      \
-  }
+  }                                                                      \
+  static_assert(true, "")
```

Comment:
nice, didn't know this trick :) 

can you add comment to explain what this things does (so that the next person won't inadvertently decide to just remove this?)

Reply:
Updated. Thanks.